.evergreen/.evg.yml

```diff
@@ -325,6 +325,21 @@ functions:
         file="${PROJECT_DIRECTORY}/.evergreen/install-dependencies.sh"
         [ -f ${file} ] && sh ${file} || echo "${file} not available, skipping"
 
+  "run perf tests":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          PROJECT_DIRECTORY=${PROJECT_DIRECTORY} .evergreen/run-perf-tests.sh
+
+  "send dashboard data":
```

Comment:
This is the way to interact with the performance graphing feature of Evergreen.  The benchmark suite will write the src/results.json file to this location.